nautobot/docs/user-guide/administration/upgrading/from-v1/ipam/index.md

```diff
@@ -0,0 +1,9 @@
+# Nautobot 2.0 IPAM Migration Guide
+
+## Goal
+
+This document aims to answer the question "What do I need to do to my data (or what changes with my data) when migrating to 2.0, specifically IPAM?"
+
+Each change is covered in detail to describe the changes, their impact, and the underlying reasons why the changes were made.
+
+Please see the individual sections for more details.
```

Comment:
Redundant to link them here too?

Reply:
As in including the TOC for this page here?

Reply:
Not for this page but the sub-pages it refers to. If we're only going to view it in the context that it has the sidebar for the associated pages, fine with me.

Reply:
I tried to make this work out of the box and it's not possible. It's kind of annoying the way sub-trees work (or not) in MkDocs without extra plugins. As it is right now, you either have sections or pages and never the twain shall meet. To have the sub-pages linked here, it would have to be manual. Certainly not impossible, just tedious.

Reply:
<img width="559" alt="Cursor_and_IPAM_Migration_Guide_-_Nautobot_Documentation" src="https://github.com/nautobot/nautobot/assets/138052/f5421135-71ea-4ab6-9160-75360b9576db">

```diff
diff --git a/nautobot/docs/user-guide/administration/upgrading/from-v1/ipam/index.md b/nautobot/docs/user-guide/administration/upgrading/from-v1/ipam/index.md
index c23c1ba85..b792aa0c0 100644
--- a/nautobot/docs/user-guide/administration/upgrading/from-v1/ipam/index.md
+++ b/nautobot/docs/user-guide/administration/upgrading/from-v1/ipam/index.md
@@ -7,3 +7,10 @@ This document aims to answer the question "What do I need to do to my data (or w
 Each change is covered in detail to describe the changes, their impact, and the underlying reasons why the changes were made.

 Please see the individual sections for more details.
+
+## Sections
+
+- [What's Changed](whats-changed.md)
+- [Before you Upgrade](before-you-upgrade.md)
+- [After you Upgrade](after-you-upgrade.md)
+- [For Developers](for-developers.md)
(END
```